src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public abstract class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer  started its work");

```

Comment:
The message in the 'doWork' method should be more informative. It should specify that it is a Bulldozer that has started working, e.g., 'Bulldozer is working.'